stable/datadog/templates/agent-clusterrole.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if and .Values.rbac.create .Values.clusterAgent.enabled -}}
+apiVersion: {{ template "rbac.apiVersion" . }}
+kind: ClusterRole
+metadata:
+  labels:
+    app: "{{ template "datadog.fullname" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+  name: {{ template "datadog.clusterAgent.fullname" . }}
+rules:
+- apiGroups:
+  - ""
+  resources:
```

Comment:
if the cluster agent is used, we could trim the clusterrole of the node agent down to:
```
- nonResourceURLs:
  - "/version"
  - "/healthz"
  verbs:
  - get
```
As the node agent only needs to communicate with the kubelet.
We would also want to add:
```
- apiGroups:
  - ""
  resources:
  - nodes
  verbs:
  - get
``` 
As the option NODE_LABELS_AS_TAGS is only supported in 6.5+. So we can add it to the list above.

As you are only using one cluster role shared by both, I'm happy to leave it as is for now, and we can make this change later.

Reply:
@CharlyF Are we ok with this one now and can merge?